src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsNumberAttendeesMapping.java

```diff
@@ -26,12 +27,24 @@ private HearingsNumberAttendeesMapping() {
     }
 
     public static int getNumberOfPhysicalAttendees(@Valid SscsCaseData caseData) {
-        int numberOfAttendees = 0;
 
         if (FACE_TO_FACE != HearingsChannelMapping.getHearingChannel(caseData)) {
```

Comment:
This if block is not covered by tests
https://sonarcloud.io/component_measures?metric=new_coverage&selected=SSCSHA%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fsscs%2Fhelper%2Fmapping%2FHearingsNumberAttendeesMapping.java&view=list&pullRequest=427&id=SSCSHA

Reply:
Added new test cases